### PR TITLE
[synthetics-job-manager] Add new runtime charts

### PR DESCRIPTION
**Pull request by sadafarshad**
#### Is this a new chart
Yes.

#### What this PR does / why we need it:
Ticket: [NR-27388](https://issues.newrelic.com/browse/NR-27388)

This PR adds the new charts for the Synthetics Job Manager with the new runtimes and these changes are needed to be able to migrate off the old helm charts to the new ones.

**Charts are getting installed and running:**
Screen Shot 2022-08-25 at 12 23 02 PM

**Running Jobs:** 
Screen Shot 2022-08-25 at 12 07 34 PM

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

**Review by sshah-nr — APPROVED** [review]

**Comment by kang-makes** [comment]
Hi!

As it is a new chart, the [`CODEOWNERS` file](https://github.com/newrelic/helm-charts/blob/81f32a5/CODEOWNERS#L10-L11) is pointing us as the owners, because it is taking the wildcard on top of it.

Please, add your team as CODEOWNERS for this new chart so you can own it and merge PRs without our approval. As one of your teammates have approved it I have no problem approving it either.

PRs fixing the spelling and removing the bumpers action have been merged so you might also want to rebase to the tip of `master` to have your tests clean.

Sorry for taking this long to answer.

File: charts/synthetics-job-manager/Chart.yaml
Comment (kaschaefer-nr) [inline charts/synthetics-job-manager/Chart.yaml:6]: this should be release-155, I think.

**Review by kaschaefer-nr — APPROVED** [review]
LGTM

**Review by kaschaefer-nr — APPROVED** [review]

File: charts/synthetics-job-manager/Chart.yaml
Comment (gsanchezgavier) [inline charts/synthetics-job-manager/Chart.yaml:22]: has `helm dependency update ...` been run for this chart? I don't see the `chart.lock` file for it, but it might be because of nested charts approach.

**Review by gsanchezgavier — APPROVED** [review]
LGTM

**Pull request merged**
